src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -275,7 +270,7 @@ private static int BioRead(SafeBioHandle bio, IntPtr buffer, int count)
             int bytes = libssl.BIO_read(bio, buffer, count);
             if (bytes != count)
             {
-                throw CreateSslException("Failed in Read BIO");
+                Debug.Fail("BIO read failed.");
```

Comment:
user of these methods does the relevant check & throws the exception. So exception not needed here.